src/language-css/parser-postcss.js

```diff
@@ -211,6 +211,14 @@ function parseNestedCSS(node) {
           value: node.params
         };
       } else {
+        if (node.params.indexOf("#{") !== -1) {
```

Comment:
`node.params.includes("#{")`?

Reply:
That’ll [have to wait](http://node.green/#ES2016-features-Array-prototype-includes) until April (or until we drop support for Node 4), unfortunately.

Reply:
I think you confuse `String.prototype.includes` and `Array.prototype.includes`. The string one is available in Node 4.